package.json

```diff
@@ -14,15 +15,23 @@
   },
   "author": "Grant Timmerman",
   "license": "MIT",
+  "engines": {
+    "node": ">=8.10.0"
+  },
   "bugs": {
     "url": "https://github.com/grant/ts2gas/issues"
   },
   "homepage": "https://github.com/grant/ts2gas#readme",
   "dependencies": {
+    "lodash": "^4.17.11",
```

Comment:
Use es6 features instead of lodash.

Reply:
OK. I just didn't want to re-invent the wheel and re-implement deep-assign (merge/mergeWith)